objects/client.c

```diff
@@ -1602,6 +1602,17 @@ luaA_client_swap(lua_State *L)
         *ref_swap = c;
 
         luaA_class_emit_signal(L, &client_class, "list", 0);
+
+        luaA_object_push(L, swap);
+        lua_pushboolean(L, true);
+        luaA_object_emit_signal(L, -4, "swapped", 2);
+        lua_pop(L, 2);
+
+        luaA_object_push(L, swap);
+        luaA_object_push(L, c);
+        lua_pushboolean(L, false);
+        luaA_object_emit_signal(L, -3, "swapped", 2);
+        lua_pop(L, 3);
```

Comment:
Not really, there is 2 clients involved in a swap, both need to be notified or it void the whole concept